recipe/meta.yaml

```diff
@@ -447,6 +451,10 @@ outputs:
       run:
         - {{ pin_subpackage('dal-include', exact=True) }}
         - {{ pin_subpackage('dal', exact=True) }}
+    test:
+      commands:
+        - ls -A $PREFIX/lib/*      # [unix]
+        - ls -A $PREFIX/include/*  # [unix]
```

Comment:
These lines are useful for debugging. Can we test the required files instead? Like:
```
- test -f $PREFIX/include/FILENAME.h   # [unix]
- test -f $PREFIX/lib/FILENAME.dylib  # [osx]
- test -f $PREFIX/lib/FILENAME.so  # [linux]
```

Reply:
Added some tests like this. Thanks, @skupr-anaconda !

Reply:
👍